src/core/ext/filters/client_channel/lb_policy/xds/xds.cc

```diff
@@ -1321,11 +1321,10 @@ void XdsLb::ProcessChannelArgsLocked(const grpc_channel_args& args) {
 
 void XdsLb::UpdateLocked(const grpc_channel_args& args) {
   ProcessChannelArgsLocked(args);
-  // If fallback is configured and the RR policy already exists, update
-  // it with the new fallback addresses.
-  if (lb_fallback_timeout_ms_ > 0 && rr_policy_ != nullptr) {
-    CreateOrUpdateRoundRobinPolicyLocked();
-  }
+  // Note: We have disable fallback mode in the code, so we dont need to update
```

Comment:
Nit: disable -> disabled. dont-> don't. "update the policy" is ambiguous here; suggest saying that ", so we don't need to handle the fallback address changes."

Reply:
done